drivers/src/sha384acc.rs

```diff
@@ -42,13 +49,13 @@ impl Sha384Acc {
     ///
     /// * `Sha384AccOp` - On, success, an object representing the SHA384 accelerator operation.
     /// * 'None' - On failure to acquire the SHA384 Accelerator lock.
-    pub fn try_start_operation(&mut self) -> Option<Sha384AccOp> {
```

Comment:
Add a comment that callers to set `assumed_lock_state` unless they are the first caller to the peripheral after boot.